### PR TITLE
Change argument names in AddSmartDisplayAd code example

### PR DESCRIPTION
**Pull request by wfansh**

### CODE REVIEW

File: examples/advanced_operations/add_smart_display_ad.pl [hunk 1]
```diff
@@ -221,23 +218,25 @@ sub create_ad_group {
...
-    $api_client, $customer_id, $ad_group_resource_name,
-    $marketing_image_asset_resource_name,
-    $square_marketing_image_asset_resource_name
-  ) = @_;
+  my ($api_client, $customer_id, $ad_group_resource_name,
+    $marketing_image_asset_id, $square_marketing_image_asset_id)
+    = @_;
 
   # Create a new image asset for marketing image and square marketing image if
   # there are no assets' resource names specified.
```
Comment (PierrickVoulet) [inline examples/advanced_operations/add_smart_display_ad.pl:226]: "there are no assets' **IDs** specified"

File: examples/advanced_operations/add_smart_display_ad.pl [hunk 2]
```diff
@@ -389,9 +382,9 @@ =head1 SYNOPSIS
 
 add_smart_display_ad.pl [options]
 
-    -help                                           Show the help message.
-    -customer_id                                    The Google Ads customer ID.
-    -marketing_image_asset_resource_name            [optional] The resource name of marketing image asset.
-    -square_marketing_image_asset_resource_name     [optional] The resource name of square marketing image asset.
+    -help                                Show the help message.
+    -customer_id                         The Google Ads customer ID.
+    -marketing_image_asset_id            [optional] The resource name of marketing image asset.
```
Comment (PierrickVoulet) [inline examples/advanced_operations/add_smart_display_ad.pl:387]: "The **ID**"

File: examples/advanced_operations/add_smart_display_ad.pl [hunk 3]
```diff
@@ -389,9 +382,9 @@ =head1 SYNOPSIS
...
+    -square_marketing_image_asset_id     [optional] The resource name of square marketing image asset.
```
Comment (PierrickVoulet) [inline examples/advanced_operations/add_smart_display_ad.pl:388]: "The **ID**"

File: examples/advanced_operations/add_smart_display_ad.pl (hunk 1 above)
Comment (wfansh) [reply examples/advanced_operations/add_smart_display_ad.pl:226]: my fault~~

File: examples/advanced_operations/add_smart_display_ad.pl (hunk 3 above)
Comment (wfansh) [reply examples/advanced_operations/add_smart_display_ad.pl:388]: good catch Pierrick. my fault

**Pull request merged**